package.json

```diff
@@ -41,63 +33,116 @@
 		"prewatch": "npm run check-engines",
 		"test": "npm run lint && npm run test-js",
 		"test-php": "cd tests && phpunit",
-		"test-js": "wp-scripts test-unit-js --config tests/javascript-config/unit/jest.config.json",
-		"test-js-ts": "jest --config config/tests/unit/jest.config.js",
-		"test-js-ts:watch": "jest --config config/tests/unit/jest.config.js --watch",
+		"test-js": "node scripts/test.js",
 		"test-js:coverage": "npm run test-js -- --coverage",
-		"test-js:coverage-ci": "npm run test-js -- --coverage --maxWorkers 1",
-		"test-js:update": "npm run test-js -- --updateSnapshot",
-		"test-js:watch": "npm run test-js -- --watch"
+		"test-js:coverage-ci": "npm run test-js -- --coverage --maxWorkers 1"
+	},
+	"browserslist": {
+		"production": [
+			">0.2%",
+			"not dead",
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
 	},
 	"husky": {
 		"hooks": {
 			"pre-commit": "lint-staged"
 		}
 	},
 	"lint-staged": {
-		"assets/ZZZ/**/*.js": [
-			"wp-scripts lint-js"
+		"assets/prototype/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+			"prettier --write",
+			"git add"
+		]
+	},
```

Comment:
This change is based on: https://create-react-app.dev/docs/setting-up-your-editor/